src/Compute/Compute.Autorest/test/utils.ps1

```diff
@@ -25,13 +26,21 @@ function setupEnv() {
     }
     set-content -Path (Join-Path $PSScriptRoot $envFile) -Value (ConvertTo-Json $env)
 
-    Import-Module Az.Compute
```

Comment:
add this back

Reply:
was able to run tests without it, once you delete generated/module/az.accounts.ps1. 

Will leave it absent